### PR TITLE
Post-submit language from David

### PR DESCRIPTION
**Pull request by atelic**

### CODE REVIEW

File: website/static/js/registrationUtils.js [hunk 1]
```diff
@@ -485,11 +485,39 @@ RegistrationEditor.prototype.submit = function() {
 						url: url,
 						data: {node: currentNode, uid: currentUser.id},
 						success: function(response) {
-							bootbox.alert("Registration submitted for review!", function(result) {
-								window.location.href = '/' + currentNode.id + '/registrations/';
+							bootbox.dialog({
```
Comment (samchrisinger) [inline website/static/js/registrationUtils.js]: @barbour-em I would like to see this text moved into a knockout template. You can use ko.renderTemplate, and inject the HTML into the bootbox modal.

An example of this is here: https://github.com/CenterForOpenScience/osf.io/blob/c8ea37a614b581a1e28daf859c9f21a35e84da04/website/static/js/metadata_1.js#L17

**Comment by atelic** [comment]
@samchrisinger Addressed comments

File: website/static/js/registrationUtils.js [hunk 2]
```diff
@@ -463,38 +471,45 @@ RegistrationEditor.prototype.create = function(schemaData) {
...
-					$.ajax({
-						method: "POST",
-						url: url,
-						data: {node: currentNode, uid: currentUser.id},
-						success: function(response) {
-							bootbox.alert("Registration submitted for review!", function(result) {
-								window.location.href = '/' + currentNode.id + '/registrations/';
-							});
+
+	bootbox.confirm(function(){
```
Comment (samchrisinger) [inline website/static/js/registrationUtils.js]: consider doing:
```
bootbox.confirm(ko.renderTemplate.bind(null, "preSubmission", Submission, {}, this,  "replaceNode"))
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind

File: website/static/js/registrationUtils.js [hunk 3]
```diff
@@ -463,38 +471,45 @@ RegistrationEditor.prototype.create = function(schemaData) {
 };
 RegistrationEditor.prototype.submit = function() {
     var self = this;
-    
+
     var currentNode = window.contextVars.node
     var currentUser = window.contextVars.currentUser
     var url = '/api/v1/project/' + currentNode.id +  '/draft/submit/' + currentUser.id + '/'
```
Comment (samchrisinger) [inline website/static/js/registrationUtils.js]: Please use `self.urls.submit`. The Editor should not construct URLs internally.

Also, this route does not need the current user's id, that is always available server-side.  So let's remove `<uid>` from the route.

In the view `user = auth.user`

File: website/static/js/registrationUtils.js (hunk 2 above)
Comment (samchrisinger) [inline website/static/js/registrationUtils.js]: please disregard

File: website/static/js/registrationUtils.js [hunk 4]
```diff
@@ -303,18 +303,26 @@ var RegistrationEditor = function(urls, editorId) {
...
     self.formattedDate = formattedDate;
-    
+
     self.iterObject = $osf.iterObject;
 
     self.extensions = {};
 };
+
+// View model for binding to templates
+var Submission = function() {
```
Comment (samchrisinger) [inline website/static/js/registrationUtils.js]: Please delete this if it's unused. 

We can pass an empty object to ko.renderTemplate.

**Comment by atelic** [comment]
@samchrisinger this should be good to go now

File: website/static/js/registrationUtils.js [hunk 5]
```diff
@@ -246,27 +246,27 @@ var Draft = function(params, metaSchema) {
...
-                    var q = self.schemaData[qid];                    
+                    var q = self.schemaData[qid];
                     if(q && !$osf.isBlank(q.value)) {
                         complete++;
                     }
                     total++;
                 });
             });
-           return Math.ceil(100 * (complete / total));
+			return Math.ceil(100 * (complete / total));
```
Comment (samchrisinger) [inline website/static/js/registrationUtils.js:261]: @barbour-em It looks like your editor is doing some reindentation/formatting of code. 

This indent for example, is wrong. 

Additionally (although non-critical), avoid reformatting comments unless they are obviously wrong.

File: website/static/js/registrationUtils.js [hunk 6]
```diff
@@ -463,37 +463,42 @@ RegistrationEditor.prototype.create = function(schemaData) {
 };
 RegistrationEditor.prototype.submit = function() {
     var self = this;
-    
+
```
Comment (samchrisinger) [inline website/static/js/registrationUtils.js:466]: In order for builds to pass on Travis, they must pass cleanly through jshint. 

This code has multiple stylistic errors. Please install and run jshint and fix the breakages.

**Comment by samchrisinger** [comment]
@barbour-em manually merging this, but please address the comments in a followup PR.

**Pull request closed without merging**